test/test_type_promotion.py

```diff
@@ -438,6 +439,29 @@ def test_transpose(self, device):
         a = torch.tensor([[True, True], [False, True]], device=device)
         self.assertEqual(a.t() == 0, a.t() == False)  # noqa: E712
 
+    @dtypes(torch.bool, torch.short, torch.uint8, torch.int, torch.long)
+    @float_double_default_dtype
+    def test_true_divide(self, device, dtype):
+        dividend = torch.randn(5, device=device).to(dtype)
```

Comment:
for non-floating dtypes, won't this basically always give me 0/1/-1?

Do we have some (testing-only) solution for this?  Like "give me X bits of noise appropriate for this dtype"?

Reply:
I'm not familiar with one. Some tests seem to have implemented their own solutions to the problem. Seems like a good separate issue?

Reply:
yes, seems like a good separate issue.  Do you mind filing an issue?